persistence/src/commonMain/db_user/com/wire/kalium/persistence/Messages.sq

```diff
@@ -577,3 +580,13 @@ markMessagesAsDecryptionResolved:
 UPDATE MessageFailedToDecryptContent
 SET is_decryption_resolved = 1
 WHERE message_id IN ?;
+
+updateMessagesTableAfterOneIsSent {
+UPDATE Message
+SET creation_date = :server_creation_date, status = 'SENT'
+WHERE id = :message_id AND conversation_id = :conversation_id;
+
+UPDATE Message
+SET creation_date = creation_date + :millis
+WHERE conversation_id = :conversation_id AND status = 'PENDING';
```

Comment:
So it's not one query, it's actually two queries in a single transaction 😅 
Should we keep it this way? I think that first query is required for us, the second one just fixes the potential issues with message order so it's a little bit less important but if if fails, it will rollback the whole transaction 🤔

Reply:
by one query, I meant one transaction with one DB connections
before there was 3 queries each waiting for a DB connection
1. update time
2. update status as SENT
3. update other pending 

3 was the slowest  since Message.status had no index

Reply:
> it will rollback the whole transaction 

i can add update or ignore for that one